src/scss/login.scss

```diff
@@ -63,9 +64,7 @@ body {
                         width: 100%;
                         border-radius: .5em;
                         height: 3em;
-
-                        &:hover {
-                            box-shadow: 2px 2px 10px rgba($color: #519188, $alpha: 0.75);
+   box-shadow: 2px 2px 10px rgba($color: #519188, $alpha: 0.75);
```

Comment:
Hier is hij niet aligned met de andere code erboven. Weet niet of dat door github komt maar miss even checken ;)

Reply:
Komt door github, heb het net gecheckt :D